tools/bazel_java_integration_test.bzl

```diff
@@ -13,6 +13,10 @@
 # limitations under the License.
 
 # Java integration test framework for using Bazel
+load(
+  "@bazel_tools//tools/build_defs/repo:http.bzl",
+  "http_archive",
+)
```

Comment:
This will break with older version of bazel.... maybe wrap it in a macro testing for native.http_archive?

Reply:
If I'm not mistaken this was released in 0.4.2 (7 Dec 2016): https://github.com/bazelbuild/bazel/commit/451599a8d22d79a84e228c9c06207df490f02f9f

The oldest version of Bazel you test against here is 0.6.0.  What is the oldest version of Bazel you'd like to support?

Reply:
You are right, my bad. The bug in the early version won't be triggered so we are good to go.